src/api/app/models/package.rb

```diff
@@ -929,8 +929,8 @@ def buildresult(prj = project, show_all = false)
   end
 
   def jobhistory_list(project, repository, arch, options = {})
-    options[:limit] = 100 if options[:limit].blank?
-    options[:package] = name if options[:package].blank?
+    options[:limit] ||= 100
+    options[:package] ||= name
```

Comment:
Why is this necessary @DavidKang ? The package name is set in the before filter? How can it be blank?

Reply:
True, it will ever exist

Reply:
@ChrisBr, we could remove it :+1:

Reply:
Then I just remove both parameters, right? 😂😅

Reply:
I think so, yes!